### PR TITLE
Rockstar for scala

### PR DESCRIPTION
**Pull request by anujbahuguna**
Created a scala rockstar version

### CODE REVIEW

File: examples/scala_rockstar.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from RockStar import RockStar
+
+scala_code = "object HelloWorld {def main(args: Array[String]) { println(\"Hello world\") }}"
```
Comment (avinassh) [inline examples/scala_rockstar.py]: Any reason you are delimiting string quotes... Following wouldn't work?

```
"object HelloWorld {def main(args: Array[String]) { println('Hello world') }}"
```

or scala expects `"`

**Comment by avinassh** [comment]
I am not sure if Scala programmers are cool enough to be rockstars.

File: examples/scala_rockstar.py (hunk 1 above)
Comment (anujbahuguna) [inline examples/scala_rockstar.py]: A valid string literal in scala has to be enclosed in double quotes else code won't compile. 
So following will result in an error during compilation : "error: unclosed character literal"
`"object HelloWorld {def main(args: Array[String]) { println('Hello world') }}"`

I wanted to have the hello world generated by rockstar script in correct syntax.

Comment (avinassh) [inline examples/scala_rockstar.py]: Yup, thats what I assumed.

**Comment by avinassh** [comment]
@anujbahuguna thanks for PR. Merged.

File: examples/scala_rockstar.py (hunk 1 above)
Comment (avinassh) [inline examples/scala_rockstar.py]: How about:

```
scala_code = 'object HelloWorld {def main(args: Array[String]) { println("Hello world") }}'
```

Comment (anujbahuguna) [inline examples/scala_rockstar.py]: Yes..above also will work. Have made changes in the pull request.

**Comment by avinassh** [comment]
Thanks anuj, merged!

**Pull request merged**